lib/pages/movie_details_page.dart

```diff
@@ -38,14 +38,14 @@ class MovieDetailPage extends StatelessWidget {
               date: result.releaseDate,
               rating: result.voteAverage.toString(),
             ),
-            MovieDetailPlayButton(),
             MovieDetailsOverview(
               overview: result.overview,
             ),
             MovieDetailsActions(),
           ],
         ),
       ),
+      bottomNavigationBar: MovieDetailPlayButton(),
```

Comment:
You can change this widget name because now is not a play button instead of you can name it something like back button

Reply:
Changed.